tools/run_tests/run_interop_tests.py

```diff
@@ -422,12 +422,18 @@ def _job_kill_handler(job):
     time.sleep(2)
 
 
-def cloud_to_prod_jobspec(language, test_case, docker_image=None, auth=False):
+def cloud_to_prod_jobspec(language, test_case, use_cloud_gateway=False, 
```

Comment:
If   using cloud gateway only means changing the server_host, we shouldn't  hard code the feature like this.  It is concievable you'll want to add more possible backends in the future and you really don't want to hand-code a boolean parameter for each of these.